react/src/types.ts

```diff
@@ -2,14 +2,15 @@ import { Maybe } from 'graphql/jsutils/Maybe';
 
 /* typescript types that map to graphql types, should be updated whenever schema is updated -- note that these are coming from our own server now */
 export interface VariantResponseInfoFields {
+    af?: Maybe<number>;
     aaAlt?: Maybe<string>;
     aaPos?: Maybe<string>;
     aaRef?: Maybe<string>;
-    cdna?: Maybe<string>;
+    cdna?: Maybe<number>;
```

Comment:
I'm not sure if `cdna` is always of type `string`. At least for gnomad, if you use panda to get back unique values for cdna, I recall sometimes seeing 21312-12312 which is a string.

Reply:
The rest of the annotation logic looks good to me!